robottelo/ui/computeresource.py

```diff
@@ -143,9 +143,125 @@ def update(self, name, newname=None, parameter_list=None,
 
     def delete(self, name, really=True):
         """Removes the compute resource entity"""
+        Navigator(self.browser).go_to_compute_resources()
```

Comment:
We do not use direct call to `Navigator`, but use `navigate_to_entity` method instead.
Anyway, you don't need that line as `self.delete_entity()` -> `self.search(name)` -> `self.navigate_to_entity()`
So remove it